frontend/src/components/Table/Table.scss

```diff
@@ -87,7 +88,7 @@
 
     .filter-wrapper {
       &.active {
-        color: #007bff;
+        color: $icon-light-color;
```

Comment:
icon light color is #3b99fc, was this intentional?

Reply:
active color is $active-color: #007bff;

Reply:
ah woops, I'll double check all of these now